src/main/java/org/prebid/server/settings/FileApplicationSettings.java

```diff
@@ -48,7 +48,7 @@ private FileApplicationSettings(SettingsFile settingsFile, Map<String, String> s
                                     Map<String, String> storedIdToImp) {
         accounts = toMap(settingsFile.getAccounts(),
                 Function.identity(),
-                account -> Account.of(account, null));
+                account -> Account.fromPriceGranularity(account, null));
```

Comment:
`FileApplicationSettings` should support new configuration values too so we should update settings file schema as well